python/lsst/sims/catUtils/baseCatalogModels/snModels.py

```diff
@@ -0,0 +1,56 @@
+"""
+The class SNObj is a catalogDB class which can read a table of SALT2
+parameters on the catsim database
+"""
+import numpy
+from .BaseCatalogModels import BaseCatalogObj
+from lsst.sims.utils import ObservationMetaData
+from lsst.sims.catalogs.db import CompoundCatalogDBObject
+
+__all__ = ['SNDBObj'] 
+
+class SNDBObj(BaseCatalogObj):
+    objid = 'TwinkUnlensedSN'
+    # From now on the tableid should be specified in instantiating the class
```

Comment:
There is only one table for other objects on fatboy, so this is specified by default (and perhaps can be changed as an attribute). We need to be able to choose which table to use since we already have two tables and will probably add a third for Twinkles3. 

It seems right to be able to specify this in the constructor.